app/graphql/graphql_devise/mutations/resend_confirmation.rb

```diff
@@ -0,0 +1,30 @@
+module GraphqlDevise
+  module Mutations
+    class ResendConfirmation < Base
+      argument :email,        String, required: true, prepare: ->(email, _) { email.downcase }
+      argument :redirect_url, String, required: true
+      
+      field :success, Boolean, null: false
+      field :message, String, null: false
+      
+      def resolve(email:, redirect_url:)
+        resource = controller.find_resource(:uid, email)
+
+        if resource
+          yield resource if block_given?
+          resource.send_confirmation_instructions({
+            redirect_url: redirect_url,
+            template_path: ['graphql_devise/mailer']
+          })
+
+          {
+            success: true,
+            message: I18n.t('graphql_devise.confirmations.send_instructions', email: email)
```

Comment:
I wasn't sure which string we should use here. There were some I grabbed from DTA's `en.yml` also. You'll see them all below and we can choose which one you guys think is best. I'm wondering if we should rely on DTA and/or Devise locales or if we should assume the user(developer) might not have those files? It might be needed that we should rely solely on our own locale strings.